index.js

```diff
@@ -25,14 +26,18 @@ app.post('/', (req, res) => {
     }
     const slashCommand = req.body.command;
     switch (slashCommand) {
+        case '/ww':
         case '/werewolf':
             const requestBody = req.body;
             const commandArray = requestBody.text.split(' ');
             if (commandArray.length) {
                 switch (commandArray[0]) {
-                    case 'new':
+                    case 'poll':
```

Comment:
We'll need to make sure to communicate this change to all the players

Reply:
Yes, how do we want to handle change logs?